advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring Commvault Express'
+---
+
+Implementing Commvault Express with EDB Postgres Advanced Server requires the following components:
+
+- EDB Postgres Advanced Server
+- Commvault Express software
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server (EPAS) instance.
+- Commvault Express installed.
+- Obtain the EDB Postgres Advanced Server (EPAS) application path and library directory path.
+- Obtain the credentials that are used to access the EDB Postgres Advanced Server (EPAS).
+- Configure the EDB Postgres Advanced Server (EPAS) archive log directory.
+
+## Configure Commvault Express for EDB Postgres Advanced Server (EPAS)
+
+### Core Setup Wizard
+1. Run the Core Setup Wizard from Commvault Express's Command Centre, the wizard helps to set up a disk storage pool and to modify the server backup plan according to your requirements.
+2. Setup Storage Pool/Disk Storage: From the Welcome page, click Let's get started. On the Disk tab, in the Name box, enter a name for the storage pool.
```

Comment:
Anything that involves clicking on the screen (i.e. going to a menu option) should be quoted as if it is code syntax.